sdk/communication/azure-communication-callautomation/src/test/java/com/azure/communication/callautomation/CallAutomationAsyncClientUnitTests.java

```diff
@@ -85,7 +86,7 @@ public void answerCallWithResponse() {
         assertNotNull(answerCallResult);
         assertEquals(200, answerCallResult.getStatusCode());
         assertNotNull(answerCallResult.getValue());
-        assertEquals("mediaSubscriptionId", answerCallResult.getValue().getCallConnectionProperties().getMediaSubscriptionId());
+        assertEquals("mediaSubscriptionId", null);
```

Comment:
you are checking for null, but your `answerCallWithResponse` seems to be receiving MediaStreamingConfiguration. that doesn't seem to be accurate.